pennylane/transforms/intermediate_reps/rowcol.py

```diff
@@ -12,13 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """CNOT routing algorithm ROWCOL as described in https://arxiv.org/abs/1910.14478."""
```

Comment:
Is there a reason ROWCOL is in caps? :eyes: 

It's RowCol in the compilation hub.